tests/unit/h/services/search_index_test.py

```diff
@@ -293,6 +293,7 @@ def test_if_the_annotation_isnt_in_the_DB_it_deletes_the_job_from_the_queue(
         job = factories.SyncAnnotationJob(annotation=annotation)
         queue_service.get.return_value = [job]
         db_session.delete(annotation)
+        db_session.commit()
```

Comment:
Need to commit for the object to be actually deleted and the test to see it gone.